WordPress/gutenberg
# Comment Reply Link: Add border support

## What?

Add border block support to the `Comment Reply Link` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Comment Reply Link` block is missing border support.

## How?

Adds the border block support in block.json.

## Testing Instructions

- Go to Global Styles Settings. ( Under Appearance > Editor > Styles > Edit styles > Blocks )
- Make sure that `Comment Reply Link` block's border is Configurable via Global Styles.
- Edit Template/Page &  Add  `Comment Reply Link` block and Apply the border Styles.
- Verify that `Comment Reply Link` block styles take precedence over global Styles.
- Verify that `Comment Reply Link` block borders display correctly in both the Editor and Frontend.

## Screenshots or Screencast 


https://github.com/user-attachments/assets/51b5f6e3-b807-4bda-9f88-1e0cc836fd7a
